inc/Plugin.php

```diff
@@ -423,6 +415,14 @@ private function init_common_subscribers() {
 			'post_subscriber',
 		];
 
+		// Only add tracking service provider if cURL extension is loaded.
+		// MixPanel (used by TrackingServiceProvider) requires cURL and will throw a fatal error if not available.
+		// This prevents crashes when WP Rocket is installed on servers without cURL support.
+		if ( function_exists( 'curl_init' ) ) {
+			$this->container->addServiceProvider( new TrackingServiceProvider() );
+			$common_subscribers[] = 'tracking_subscriber';
+		}
```

Comment:
The curl condition is no longer necessary because the mixpanel library now uses WP HTTP methods directly